missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/Arenas.java

```diff
@@ -34,10 +35,10 @@
 
 public class Arenas {
 
-    @Getter private static final List<Arena> arenas = new ArrayList<>();
+    @Getter private static final HashMap<String, Arena> ARENAS = new HashMap<>();
```

Comment:
We always use interfaces to work with, if possible.
```suggestion
    @Getter private static final Map<String, Arena> ARENAS = new HashMap<>();
```